arviz/stats/diagnostics.py

```diff
@@ -17,7 +17,7 @@
 from .stats_utils import stats_variance_2d as svar
 from .stats_utils import wrap_xarray_ufunc as _wrap_xarray_ufunc
 
-__all__ = ["bfmi", "ess", "rhat", "mcse"]
+__all__ = ["bfmi", "ess", "rhat", "mcse", "psens"]
```

Comment:
In addition to adding it here, as a new function it also needs to be added to https://github.com/arviz-devs/arviz/blob/main/doc/source/api/diagnostics.rst. After doing that and pushing, you will be able to see the rendered docs for the function at https://arviz--2093.org.readthedocs.build/en/2093/api/diagnostics.html